server/methods/requestDataDownload.js

```diff
@@ -27,6 +27,10 @@ Meteor.methods({
 			}
 		}
 
+		if (!fs.existsSync(tempFolder)) {
```

Comment:
any reason to use sync functions? why not using async ones?

Reply:
I think he is keeping it consistent with the code below this. All of them currently use the `sync` versions.